factorio

```diff
@@ -355,7 +355,7 @@ function stop_service() {
     fi
 
     # Start the reader (in case tail stopped already)
-    cat "${FIFO}" &
```

Comment:
I'm pretty sure this is here for a reason (going back the repo history may very well shed some light on it but I can not accept the pull request without digging into it further).

Reply:
I'm more then happy to accept the .gitignore change for vs code users if you care to submit a rebased single commit that does not remove the runaway tail.